<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>460</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Your finance manager</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>151</width>
      <height>390</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="categories_lable">
       <property name="text">
        <string>Categories</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QListWidget" name="categories_list">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>140</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="periond_label">
       <property name="text">
        <string>Period</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="periond_start">
       <property name="displayFormat">
        <string>M/dd/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="periond_finish">
       <property name="displayFormat">
        <string>M/dd/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="show_query_button">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="user_label">
       <property name="text">
        <string>UserName</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="change_user_button">
       <property name="text">
        <string>Change User</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="remove_category_button">
       <property name="text">
        <string>Remove category</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="items_table">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_item_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_export_to_excel"/>
    <addaction name="action_import_from_excel"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_export_to_excel">
   <property name="text">
    <string>Export to Excel</string>
   </property>
  </action>
  <action name="action_import_from_excel">
   <property name="text">
    <string>Import from Excel</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
